test/integration/targets/docker_network_facts/tasks/main.yml

```diff
@@ -0,0 +1,60 @@
+---
+- block:
+  - name: Create random network name
+    set_fact:
+      nname: "{{ 'ansible-test-%0x' % ((2**32) | random) }}"
+
+  - name: Make sure network is not there
+    docker_network:
+      name: "{{ nname }}"
+      state: absent
+      force: yes
+
+  - name: Inspect a non-present network
+    docker_network_facts:
+      name: "{{ nname }}"
+    register: result
+
+  - assert:
+      that:
+      - "not result.exists"
+      - "'docker_network' in result"
+      - "not result.docker_network"
```

Comment:
You should test here more explicitly for `{}` or `None` (whatever you decide to actually return).